### PR TITLE
Feature/sw 86 (send welcome email api)

### PR DESCRIPTION
**Pull request by dangvu2b**

### CODE REVIEW

File: modules/api/src/assets/emails/welcome-config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "from": "dangvh@2-b.io",
+  "cc": "dang.vu@ntq-solution.com.vn",
+  "subject": "Sending with Twilio SendGrid demo",
+  "text": "and easy to do anywhere, even with Node.js"
+}
```
Comment (longlh) [inline modules/api/src/assets/emails/welcome-config.json:6]: Add linebreak

File: modules/api/example.env [hunk 2]
```diff
@@ -6,3 +6,4 @@ AMQ_QUEUE=job-queue
 AMQ_PREFIX=shopify-dev.
 
 SOCKET_URL=http://localhost:3002
+SENDGRID_API_KEY=
```
Comment (longlh) [inline modules/api/example.env:9]: EMAIL_CONFIG_FILE_PATH=...

File: modules/api/src/assets/emails/welcome-config.json (hunk 1 above)
Comment (longlh) [inline modules/api/src/assets/emails/welcome-config.json:6]: move html template to this file

File: modules/api/src/services/email/sendgrid.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import config from '@/infrastructure/config'
+import SendGridEmail from '@/infrastructure/sendgrid'
+import ect from 'ect'
+
+const fs = require('fs')
+
+const getEmailConfig = (fileName) => {
```
Comment (longlh) [inline modules/api/src/services/email/sendgrid.js:7]: read config when server starts

File: modules/api/src/assets/emails/welcome-config.json (hunk 1 above)
Comment (dangvu2b) [reply modules/api/src/assets/emails/welcome-config.json]: Added

Comment (dangvu2b) [reply modules/api/src/assets/emails/welcome-config.json]: moved html template into welcome-config.json

File: modules/api/src/services/email/sendgrid.js (hunk 3 above)
Comment (dangvu2b) [reply modules/api/src/services/email/sendgrid.js]: moved readFileSync into config.js in infrastructure

File: modules/api/example.env (hunk 2 above)
Comment (dangvu2b) [reply modules/api/example.env:9]: Added

**Pull request merged**